servers/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -15,20 +15,111 @@
 
 package tachyon.worker;
 
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.server.TThreadPoolServer;
+import org.apache.thrift.transport.TServerSocket;
+import org.apache.thrift.transport.TTransportException;
+import org.apache.thrift.transport.TTransportFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.base.Throwables;
+
 import tachyon.Constants;
+import tachyon.Version;
+import tachyon.conf.TachyonConf;
+import tachyon.metrics.MetricsSystem;
+import tachyon.security.authentication.AuthenticationUtils;
+import tachyon.thrift.BlockWorkerClientService;
+import tachyon.thrift.WorkerNetAddress;
```

Comment:
since this is a refactor PR, we should probably limit the scope of this PR by just making it moving code from one class to the other. 

Using `WorkerNetAddress` requires functionality change of code. We could probably address this in separate PRs (if necessary).